bootloader/bin/bootstrap.sh

```diff
@@ -129,21 +129,23 @@ print_selector() {
   echo "│ Shimboot OS Selector │"
   echo "└──────────────────────┘"
 
-  if [ "${rootfs_partitions}" ]; then
+  # Check if there are bootable partitions
+  if [ -n "${rootfs_partitions}" ]; then
+    # Process each rootfs partition entry
     for rootfs_partition in $rootfs_partitions; do
-      #i don't know of a better way to split a string in the busybox shell
-      local part_path=$(echo $rootfs_partition | cut -d ":" -f 1)
-      local part_name=$(echo $rootfs_partition | cut -d ":" -f 2)
+      # Use IFS to split the string, separating by `:`
+      IFS=':' read -r part_path part_name _ <<< "$rootfs_partition"
       echo "${i}) ${part_name} on ${part_path}"
-      i=$((i+1))
+      i=$((i + 1))
     done
   else
-    echo "no bootable partitions found. please see the shimboot documentation to mark a partition as bootable."
+    echo "No bootable partitions found. Refer to the Shimboot documentation to mark a partition as bootable."
   fi
 
-  echo "q) reboot"
-  echo "s) enter a shell"
-  echo "l) view license"
+  echo "Options:"
+  echo "  q) Reboot"
+  echo "  s) Enter a shell"
+  echo "  l) View license"
```

Comment:
Kind of pointless to change the formatting of this. Also, the actual bootable partitions listed above this text are also valid options so putting the "Options:" text below that suggests that they are not.

Reply:
Yeah, i guess